microsite/pages/en/community.js

```diff
@@ -17,125 +41,120 @@ const Background = props => {
       <Block small className="stripe-bottom bg-black-grey">
         <Block.Container style={{ justifyContent: 'flex-start' }}>
           <Block.TextBox>
-            <Block.Title>Backstage Community</Block.Title>
-          </Block.TextBox>
-          <Block.TextBox>
+            <Block.Title main>Backstage Community</Block.Title>
             <Block.Paragraph>
-              What's the use of having fun if you can't share it? Exactly. Join
-              the vibrant community around the Backstage project. Be it on
-              GitHub, social media, Discord... You'll find a welcoming
-              environment. To ensure this, we follow the{' '}
+              Join the vibrant community around Backstage through social media
+              and different meetups. To ensure you have a welcoming environment,
```

Comment:
```suggestion
              and different meetups. To ensure that you have a welcoming environment,
```

Reply:
Either one? not sure if that reads better?

Reply:
The copy is all written in the second person, so we should go for the 'you'. "We" also poses ambiguity challenges in this case.